_posts/en/newsletters/2023-02-22-newsletter.md

```diff
@@ -0,0 +1,265 @@
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #24149][] adds signing support for P2WSH-based
+  [miniscript][topic miniscript] [output descriptors][topic
+  descriptors]. Bitcoin Core will be able to sign any miniscript
+  descriptor input if all the necessary preimages and keys are
+  available and timelocks are satisfied. Some features are still
+  missing for full Miniscript support in the Bitcoin Core wallet: the
+  wallet cannot yet estimate the input weight for some descriptors
+  before signing and cannot yet sign [PSBTs][topic PSBT] in some
+  edge-cases. Miniscript support for P2TR outputs is also still
+  pending.
+
+- [Bitcoin Core #25344][] updates the `bumpfee` and `psbtbumpfee` RPCs
+  for using [Replace By Fee][topic rbf] (RBF) to allow specifying a
+  different set of outputs than the transaction being replaced.  This
+  can be used to add new outputs (e.g. for iterative [payment
+  batching][topic payment batching]) or to remove outputs (e.g. for
+  attempting to cancel an unconfirmed payment).
+
+- [Eclair #2596][] limits the number of times a peer attempting to open
+  a [dual funded][topic dual funding] channel can [RBF][topic rbf] fee
+  bump the channel open transaction before the node won't accept any
+  further attempted updates.  This is motivated by the node needing to
+  store data about all possible versions of the channel open
+  transaction, so allowing unlimited fee bumps could be a problem.
+
+- [Eclair #2595][] continues the project's work on adding support for
+  [splicing][topic splicing], in this case with updates to the functions
+  used for constructing transactions.
+
+- [Eclair #2479][] adds support for paying [offers][topic offers].  A
+  user receives an offer, tells Eclair to pay it, Eclair uses the offer
+  to fetch an invoice from the receiver, verifies the invoice contains
+  the expected parameters, and pays the invoice.
+
+- [LND #5988][] adds a new optional probability estimator for finding
+  payment paths.  It is partly based on previous pathfinding research
+  (see [Newsletter #192][news192 pp]) with additional input from other
+  approaches.
+
+- [Rust Bitcoin #1636][] adds a `predict_weight()` function that accepts
+  information about a transaction to construct and returns the expected
+  weight of the transaction.  This is especially useful for fee
+  management: to determine which inputs need to be added to a
+  transaction, the amount of the fee needs to be known, but to determine
+  the amount of the fee, the size of the transaction needs to be known.
+  The function can provide a reasonable size estimate without actually
+  having to construct a candidate transaction.
```

Comment:
Mh, they’re part of the Lucky 10,000 today. I just wrote them a few lines about using the _effective values_ of UTXOs during coin selection.